common.mk

```diff
@@ -142,15 +141,34 @@ FIRTOOL_TARGETS = \
 	$(FIRTOOL_SMEMS_CONF) \
 	$(FIRTOOL_TOP_SMEMS_JSON) \
 	$(FIRTOOL_TOP_HRCHY_JSON) \
-	$(FIRTOOL_MODEL_MOD_HRCHY_JSON) \
+	$(FIRTOOL_MODEL_HRCHY_JSON) \
 	$(FIRTOOL_MODEL_SMEMS_JSON) \
 	$(FIRTOOL_FILELIST) \
 	$(FIRTOOL_BB_MODS_FILELIST)
 
+ifeq (,$(ENABLE_CUSTOM_FIRRTL_PASS))
```

Comment:
I would add a comment here describing the intent of the variables (aka what does the repl-seq-mem pass, etc)